proxy/multi_app_conn_test.go

```diff
@@ -0,0 +1,80 @@
+package proxy
+
+import (
+	"errors"
+	"os"
+	"os/signal"
+	"syscall"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/mock"
+
+	abcimocks "github.com/tendermint/tendermint/abci/client/mocks"
+	"github.com/tendermint/tendermint/proxy/mocks"
+)
+
+func TestAppConns_Start_Stop(t *testing.T) {
+	quitCh := make(<-chan struct{})
+
+	clientCreatorMock := &mocks.ClientCreator{}
+
+	clientMock := &abcimocks.Client{}
+	clientMock.On("SetLogger", mock.Anything).Return()
+	clientMock.On("Start").Return(nil)
+	clientMock.On("Stop").Return(nil)
+	clientMock.On("Quit").Return(quitCh)
+
+	clientCreatorMock.On("NewABCIClient").Return(clientMock, nil).Times(4)
+
+	appConns := NewAppConns(clientCreatorMock)
+
+	appConns.Start()
+	clientMock.AssertCalled(t, "Start")
+
+	appConns.Stop()
+	clientMock.AssertCalled(t, "Stop")
+}
+
+// Upon failure, we call tmos.Kill
+func TestAppConns_Failure(t *testing.T) {
```

Comment:
not sure how useful these tests are actually (cc @marbar3778 )

Reply:
some tests are better than no tests 😄 they seem simple enough that they may not be useful but helps to have them just incase. 

it doesnt hurt to have them right?

Reply:
I guess, yeah